content/issue-2/how-i-build-backon.md

```diff
@@ -0,0 +1,187 @@
+[backon](https://github.com/Xuanwo/backon) is a Rust error retry library, and this article aims to share some techniques I used in implementing it~
+
+```urlpreview
+https://github.com/Xuanwo/backon
+```
+
+## Origin
+
+When implementing the RetryLayer for [OpenDAL](https://github.com/datafuselabs/opendal), I needed to provide a backoff mechanism to implement features such as exponential backoff and jitter. Although I found [`backoff`](https://github.com/ihrwein/backoff) through a simple search, I wasn't quite satisfied. First, I noticed that the maintenance status of the library didn't seem to be good, with 4 unmerged PRs, and the main branch was last updated in 2021. Secondly, I didn't like the API it provided:
```

Comment:
Would be better if we can give some explanation to help the readers understand what is `backoff`, and or maybe give a good article link about `backoff`. 👍

Reply:
Done